client/src/views/Environment.tsx

```diff
@@ -19,51 +20,69 @@ const ENDPOINTS_POLLING_INTERVAL = 5000;
 
 function Environment() {
   const theme = useTheme();
-  const { output, environment, stopEnvironment, relaunchEnvironment } = useOkteto();
+  const { output, environment, status, stopEnvironment, currentContext, relaunchEnvironment } = useOkteto();
   const [endpoints, setEndpoints] = useState<Array<string>>([]);
+  const prevStatusRef = useRef<OktetoStatus | null>(null);
+
 
   const handleOpenEnvironment = () => {
     if (environment) {
       window.ddClient.host.openExternal(environment.link);
     }
   };
 
+  useEffect(() => {
+    if ( prevStatusRef.current && prevStatusRef.current === 'activating' && status === 'ready')
```

Comment:
In a first pass I think that since the status is checked via a polling executed in an interval of time, we might hit the case where the activation happens so fast that the value of status doesn't necessarily transition from 'activating' ->  'ready', so it might not trigger the notification. We could just check the transition to "ready" from any other state, no "activating".

Reply:
That looks good. I'll be testing it a bit, with the status polling this condition can be tricky in some corner cases. Once all is good, we will merge it. Thanks a lot for your contribution 👏 👏 👏